UnityProject/Assets/Scripts/Messages/Client/Admin/RequestUpdatePlayerCharacterSheet.cs

```diff
@@ -0,0 +1,109 @@
+﻿using System;
+using Logs;
+using Messages.Server;
+using Mirror;
+using Newtonsoft.Json;
+using Systems.Character;
+using UI.Admin.DIMGUI.Characters;
+using UnityEngine;
+
+namespace Messages.Client.Admin
+{
+	public class RequestUpdatePlayerCharacterSheet : ClientMessage<RequestUpdatePlayerCharacterSheet.NetMessage>
+	{
+		public struct NetMessage : NetworkMessage
+		{
+			public string CharacterSheetJson;
+			public string AccountID;
+		}
+
+		public override void Process(NetMessage msg)
+		{
+			if (IsFromAdmin() == false)
+			{
+				Loggy.Error($"Lacking admin permissions on {SentByPlayer.Username}.");
```

Comment:
use format when you are formatting text. This is meaningless now, but it is best practices and it will come in handy when/if we move to another more robust logging solution that structures the logs.

```suggestion
				Loggy.Error().Format($"Lacking admin permissions on {SentByPlayer.Username}.");
```

Reply:
why would this need an entire function? why can't we just pass interpolated strings immediately?

Reply:
ah, i see why

Reply:
like I said, when/if we ever transition to a more robust logging solution like serilog, using template strings separated from the data allows you to have more meaningful structured logs. You could then do cool stuff like for example query any log where ``username`` is ``MaxIsjoe`` in your logs reader.

Idea is passing data to log, not plain strings.

Reply:
that being said, if we never actually do that then it has no purpose.